### DISCUSSION TITLE
The B1 Propellers aren't fitting properly.

### DISCUSSION THREAD

**Discussion by akshaysunkara** [bitcraze/discussions#1380] [Q&A]
Hey guys, for some reason my B1 propellors aren't fitting correctly. I have three types of propellors right now:

1) The A propellors
2) the B propellors
3) some transperent propellers (it says HQProp,  51-MMr9 on it)

but the b propellors aren't fitting for some reason. what do i do? could i use the transparent propellers instead?

**Comment by akshaysunkara** [comment]
I also can't use the transperent propeller on the motors since they're too big

**Reply by akshaysunkara** [reply]
if i need to order propellors, which ones do i need to buy?

**Reply by gemenerik** [reply]
It depends on what motors you are using. From what you are saying, it sounds like you are using the stock CF2.1 motors (7x16mm), and thus would need [the stock CF2.1 propellers](https://store.bitcraze.io/products/propeller-pack). But please verify.

**Reply by akshaysunkara** [reply]
Okay, that makes sense, thank you.
